packages/space-header/src/index.ts

```diff
@@ -23,10 +23,16 @@ async function main(initialSpace: string | Space, options?: Options) {
 	if (typeof initialSpace === "string") {
 		space = await get_space(initialSpace);
 		if (space === null) return console.error("Space not found");
+	} else {
+		space = initialSpace;
 	}
```

Comment:
There was a bug with passing the space information directly, we checked the  `initialSpace` value but never assigned it to there `space` variable if it isn't a string. This is what gets passed along, so this mechanism wasn't working previously.